cmd/ip-masq-agent/ip-masq-agent.go

```diff
@@ -239,6 +242,12 @@ func (c *MasqConfig) validate() error {
 			return fmt.Errorf("ipv6 is not enabled, but ipv6 cidr %s provided. Enable ipv6 using --enable-ipv6 agent flag", cidr)
 		}
 	}
+	// check if SNAT target is a valid ip
+	if c.SnatTarget != "" {
```

Comment:
```
if c.SnatTarget != "" && net.ParseIP(c.SnatTarget) == nil {
  return fmt.Errorf("SnatTarget is not a valid IP (%q)", c.SnatTarget)
}
```

Reply:
thanks for the suggestion. I was following the same method we are using to validateCIDR but happy to make this shorter